modules/auth/schema/creation.go

```diff
@@ -56,6 +56,9 @@ func (s *Schema) SchemaCreation(ctx context.Context, dbType, col, project, schem
 			if err := checkErrors(realFieldStruct); err != nil {
 				return err
 			}
+			if realFieldStruct.IsList { // as directive is relation for array type don't generate queries
```

Comment:
not needed 
as we have called check error func which make sure's that array without type relation is not allowed
so if the above code has come for execution it means the array is of type directive
otherwise, check error func would have given error

Reply:
I know, but let the code be self explanatory. No one will actually go through `checkType` to understand that this is only for relations